package.json

```diff
@@ -93,13 +94,17 @@
     "watch": "yarn install && BABEL_ENV=dev NODE_ENV=dev ./node_modules/.bin/webpack --watch --progress --colors --config build/webpack.config.js",
     "test": "yarn install && NODE_ENV=test ./node_modules/.bin/karma start build/karma.conf.js",
     "debug": "yarn install && NODE_ENV=test ./node_modules/.bin/karma start build/karma.conf.js --no-single-run --auto-watch",
+    "setup-travis": "cd functional-tests && yarn install && node app.js",
```

Comment:
does this need to point to the functional test folder as well? `functional_tests/app.js`

Reply:
it cd's into the functional test folder so it's fine